common/supplemental/supplementalData.xml

```diff
@@ -2735,9 +2735,9 @@ XXX Code for transations where no currency is involved
 		</territory>
 		<territory type="CH" gdp="733800000000" literacyPercent="99" population="8860570">	<!--Switzerland-->
 			<languagePopulation type="de" populationPercent="73" officialStatus="official"/>	<!--German-->
+			<languagePopulation type="fr" populationPercent="67" officialStatus="official" references="R1335"/>	<!--French-->
```

Comment:
This is very high. While living in Switzerland (Zürich), my experience was that just small percentage of German Swiss had sufficient French to be comfortable using it in UIs. Many had some years in school, but that didn't translate into adult competency. When German Swiss had conversations with French Swiss, it was far more likely to be in English. Now, it is likely a higher percentage in the cantons nearest (or split across) Romandy, I just don't see the figures adding to 67%.

Not at all hard evidence, but an AI responded 

"The information regarding the percentage of the Swiss population who speak fluent French or can get by with the language comes from a combination of sources, including:

Swiss Federal Statistical Office:

 This official government agency collects and publishes data on various aspects of Swiss society, including language use. Their surveys and censuses provide reliable information on the language proficiency of the population.   

Academic Research and Publications: Studies conducted by linguists and sociologists specializing in Switzerland often include data on language distribution and proficiency. These studies provide a more nuanced understanding of language use in the country.

Reputable News Outlets and Online Resources: Articles and reports from trusted news sources and websites that focus on Switzerland often cite statistics on language demographics. These sources can help corroborate information from official and academic sources.

While these sources may not always perfectly align in their exact figures, they generally converge on the estimate that approximately 23% of the Swiss population speaks fluent French and roughly 28% can get by with the language, including those who are fluent.

Reply:
The French source I used expected 67%, but it was counting L2 speakers, not strictly filtered by literacy. FWIW most of my Swiss friends (from either side of the border) can speak French but I wouldn't consider them a representative sample. I can look for other sources for Switzerland though since they will be very thorough.